### PR TITLE
docs: `integrations` updates 21

### PR DESCRIPTION
**Pull request by leo-gan**
Added missed provider pages. Added descriptions and links. Fixed inconsistency in text formatting.

### CODE REVIEW

File: docs/docs/integrations/providers/milvus.mdx [hunk 1]
```diff
@@ -9,17 +9,25 @@
 Install the Python SDK:
 
 ```bash
-pip install pymilvus
+pip install pymilvus langchain-milvus
```
Comment (efriis) [inline docs/docs/integrations/providers/milvus.mdx:12]:
```suggestion
pip install langchain-milvus
```

File: docs/docs/integrations/providers/mongodb_atlas.mdx [hunk 2]
```diff
@@ -22,10 +22,30 @@ See a [usage example](/docs/integrations/vectorstores/mongodb_atlas).
...
+## Retrievers
 
-## LLM Caches
+### Full Text Search Retriever
+
+>`Hybrid Search Retriever` performs full-text searches using 
+> Lucene’s standard (`BM25`) analyzer.
+
+```python
+from langchain_mongodb.retrievers.full_text_search import MongoDBAtlasFullTextSearchRetriever
```
Comment (efriis) [inline docs/docs/integrations/providers/mongodb_atlas.mdx:33]:
```suggestion
from langchain_mongodb.retrievers import MongoDBAtlasFullTextSearchRetriever
```

File: docs/docs/integrations/providers/mongodb_atlas.mdx [hunk 3]
```diff
@@ -22,10 +22,30 @@ See a [usage example](/docs/integrations/vectorstores/mongodb_atlas).
...
+```
+
+### Hybrid Search Retriever
+
+>`Hybrid Search Retriever` combines vector and full-text searches weighting 
+> them the via `Reciprocal Rank Fusion` (`RRF`) algorithm.
+ 
+```python
+from langchain_mongodb.retrievers.hybrid_search import MongoDBAtlasHybridSearchRetriever
```
Comment (efriis) [inline docs/docs/integrations/providers/mongodb_atlas.mdx:42]:
```suggestion
from langchain_mongodb.retrievers import MongoDBAtlasHybridSearchRetriever
```

**Comment by leo-gan** [comment]
@efriis Thanks!

**Pull request merged**